test/WebJobs.Script.Tests/PreJit/JitReadTest.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Middleware;
+using Microsoft.Diagnostics.JitTrace;
+using Microsoft.WebJobs.Script.Tests;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.PreJIT
+{
+    [Trait(TestTraits.Category, TestTraits.EndToEnd)]
+    [Trait(TestTraits.Group, TestTraits.ReleaseTests)]
+    public class JitReadTest
```

Comment:
please add this test in the same test file - [JitPreparesTest.cs]

Reply:
Got it. So, both tests should run only during releases?

Reply:
possible to apply ReleaseTests trait at test level?